### PR TITLE
Add auto-rendered product layout

### LINKED ISSUE #128: Add auto-rendered product layout
Add auto-rendered product layout where the product data is rendered based on [product drop](https://www.voog.com/developers/markup/ecommerce-objects/product) data.

It should look like current "Product" layout. Use new [image](https://www.voog.com/developers/markup/tags/image) or [image_data](https://www.voog.com/developers/markup/tags/image-data) tag for product cover image.

It content type must be `product`:

```json
{
  "content_type": "product",
  "component": false,
  "file": "layouts/auto_generated_product.tpl",
  "layout_name": "auto_generated_product",
  "title": "Auto-generated product"
}
```

See also buy button rendering example in [Nuuk design](https://github.com/Voog/design-nuuk/blob/master/components/buy-button.tpl).

## TODO

* [ ] Add "Auto-generated product" layout
* [ ] Check and improve "[product list](https://www.voog.com/support/online-store/product-list-content-area)" widget styles when needed.

### PR DESCRIPTION
**Pull request by PeeterPaal**
Adds auto-rendered product layout.
Also adds a browser hack to enhance image rendering in Chrome based browsers.
Closes #128

### CODE REVIEW

**Review by tanelj — APPROVED** [review]
Nice work!

**Review by tanelj — APPROVED** [review]
Nice work!

File: javascripts/application.js [hunk 1]
```diff
@@ -1136,6 +1136,50 @@ MMCQ = (function() {
...
+      changeProductImagePos();
+    }, 25));
+
+    var changeProductImagePos = function () {
+      var productGallery = $('.js-product-gallery');
+      var productImageContentBox = $('.js-content-item-box');
+      var productRightContent = $('.js-product-right-content');
+
+      if ($('.js-buy-btn-content .edy-buy-button-container').length >= 1) {
+        if ($(window).width() < 640) {
```
Comment (pxska) [inline javascripts/application.js]: This value should be 1px higher (641).

**Review by pxska — APPROVED** [review]
A very tiny (1px) comment. Otherwise good!

**Pull request merged**